tools/kcli/kcli/parse.py

```diff
@@ -54,9 +54,9 @@ def create_parser(options_trees):
         sub_parser.add_argument("-e", "--extra-vars", default=list(),
                                 action='append', help="Provide extra vars")
         sub_parser.add_argument("-n", "--input", action='append',
-                                help="a settings file that will be loaded "
-                                     "first, all other settings file will be"
-                                     " merged with it", default=list())
+                                help="Settings files to be loaded first,"
+                                     " other settings file will be"
+                                     " merged with them", default=list())
```

Comment:
What's the merge order/priority? will "input" files override the "options" or vice-versa?